cmake/Modules/FindBLAS.cmake

```diff
@@ -62,12 +62,12 @@ MACRO(Check_Fortran_Libraries LIBRARIES _prefix _name _flags _list)
       if ( APPLE )
         find_library(${_prefix}_${_library}_LIBRARY
           NAMES ${_library}
-          PATHS /usr/local/lib /usr/lib /usr/local/lib64 /usr/lib64 /opt/OpenBLAS/lib /usr/lib/aarch64-linux-gnu
+	  PATHS /usr/local/lib /usr/lib /usr/local/lib64 /usr/lib64 /opt/OpenBLAS/lib $HOME/amd-blis-mt/lib /usr/lib/aarch64-linux-gnu
```

Comment:
Thanks for the suggestions. We are working on making necessary changes, once complete we will make changes to our PR.

Reply:
For now, we are following this procedure to build PyTorch with MKLDNN+BLIS instead of default MKLDNN+MKL, any suggestions on how to set the cmake variable (BLAS as blis).
-----------------------
$USE_ROCM=0 python setup.py build --cmake-only
$ccmake build
Set the following environment variables from ccmake build:
BLAS as blis
-----------------------